src/people.rs

```diff
@@ -285,6 +380,20 @@ impl PeopleData {
         let mut property_ref = self.get_person_property_ref(person_id, property);
         *property_ref = Some(value);
     }
+
+    fn check_initialization_list<T: InitializationList>(
+        &self,
+        initialization: &T,
+    ) -> Result<(), IxaError> {
+        let properties_map = self.properties_map.borrow();
+        for (t, property) in properties_map.iter() {
```

Comment:
Probably worth a comment that this only checks properties that have been set at least once